packages/build/.babelrc.js

```diff
@@ -22,12 +22,11 @@ const plugins = [
 ];
 
 function makePresents(test = false) {
-  const presents = ['@babel/preset-react', '@babel/preset-typescript'];
-
   if (test) {
     return [
       ['@babel/preset-env', { targets: { node: 'current' } }],
-      ...presents,
+      '@babel/preset-react',
+      '@babel/preset-typescript',
```

Comment:
So now only tests need those babel presets because in dev/production builds React and TypeScript stuff is going to be handled by ts-loader, do I understand this right?

Reply:
Yeah, you're spot on

Reply:
Well, JSX is still handled by babel for the styled components babel plugin to work